### DISCUSSION TITLE
Убрать прокрутку карты при касании

### DISCUSSION THREAD

**Discussion by maxinflame** [yandex-maps-unofficial/vue-yandex-maps#312] [Q&A (1.x)]
Требуется сделать так, чтобы на мобильных устройствах при касании карты одним пальцем карта не прокручивалась, а продолжался обычный скролл страницы. А при касании двумя пальцами можно было бы прокручивать карту. Пробовал убирать drag из behaviors, но тогда страница перестаёт скроллиться. 
Подскажите, как такое реализовать?

**Comment by mihaon** [comment]
Тоже сейчас ищу решение данной проблемы. В ymaps 2.1 у `behaviors` можно было задать `multiTouch` для моб. устройств (которое в точности реализует описанное поведение), но в версии 3 почему-то не реализовали `multiTouch`. Кому-то удалось найти решение данной проблемы?
